### PR TITLE
Added enableAzureRBAC property in ManagedClusterAADProfile

### PR DESCRIPTION
**Pull request by Anumita**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json [hunk 1]
```diff
@@ -2222,6 +2222,10 @@
           "type": "boolean",
           "description": "Whether to enable managed AAD."
         },
+        "enableAzureRBAC": {
+          "type": "boolean",
+          "description": "Whether AAD needs to be used for authorization checks on the AKS cluster"
```
Comment (weinong) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json:2227]:
```suggestion
          "description": "Whether to enable Azure RBAC for Kubernetes authorization."
```

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json [hunk 2]
```diff
@@ -2222,6 +2222,10 @@
           "type": "boolean",
           "description": "Whether to enable managed AAD."
         },
+        "enableAzureRBAC": {
```
Comment (NullMDR) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json:2225]: Add example for this new property?

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json (hunk 2 above)
Comment (Anumita) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json:2228]: I have created a new file for the example .\examples\ManagedClustersCreate_UpdateWithEnableAzureRBAC.json

**Comment by yungezz** [comment]
hi @NullMDR is this PR ok to merge? thanks.

**Pull request merged**